pyproject.toml

```diff
@@ -171,7 +170,6 @@ disable = [
     "too-many-public-methods",
     "too-many-boolean-expressions",
     "wrong-import-order",
-    "consider-using-f-string",
```

Comment:
Why do we enable this in pylint if ruff is handling it?

Reply:
Done